### DISCUSSION TITLE
feed generate limitation

### DISCUSSION THREAD

**Discussion by gkostopoulos** [MISP/PyMISP#791] [Q&A]
Hi , 
I'm trying to generate a feed using the generate.py script and I receive an error 500 .
The script is working fine when I set the entries parameter up to 3000 on the settings.py
But if I set it to a bigger value it fails.
Is there a limitation in the feed size ?
My server is Redhat 7.9 4 cores 16GB of memory.

**Comment by Rafiot** [comment]
Can you check where it fails? I'm wondering if it fails at: https://github.com/MISP/PyMISP/blob/main/examples/feed-generator/generate.py#L60 or somewhere else.

And an other thing: please have a look at the error logs (`<MISP>/app/tmp/logs/error.log`), and let us know what it says. If it is related to the memory, please try to increase the php memory limit.

**Reply by gkostopoulos** [reply]
Thank you for the reply .
I have attached a  sample of the log files.
In principle is the same error message repeating.
I have tried to relax the php  settings with no luck.

[error.log_sample.txt](https://github.com/MISP/PyMISP/files/7278688/error.log_sample.txt)
[php.ini.txt](https://github.com/MISP/PyMISP/files/7278695/php.ini.txt)
[resque-worker-error.log.sample.txt](https://github.com/MISP/PyMISP/files/7278703/resque-worker-error.log.sample.txt)


I

**Comment by Rafiot** [comment]
This error sounds like an issue with your ssl setup, nothing to do with the memory, as far as I can tell.

I don't think this one is related to the 500 error you get with the feed export.

**Reply by gkostopoulos** [reply]
That was my first assumption as well.
I created a new certificate.Same outcome .I m using a self signed certificate. SSL verification is disabled.
When the feed export is limited to something like 3000 events I don't have an issue.

**Comment by Rafiot** [comment]
I'll need you to find the exception you're getting that is related to the error, because I'm not sure how we can help here.
